internal/command/views/hook_ui.go

```diff
@@ -35,6 +35,13 @@ func NewUiHook(view *View) *UiHook {
 	}
 }
 
+func NewUiOptionalHook(view *View) tofu.Hook {
```

Comment:
I'm curious what folks think of the name.  Go has some interesting conventions about what is and what isn't a "New"/constructor.  Other than that, this PR looks good :+1:

Reply:
Given that this is an internal API between two packages that are closely related (`views` exists primarily to support `command`), I personally think this function name is fine for now and it'll be easy to rename it later if we find that it's somehow confusing.